kubernetes/rt.yml.erb

```diff
@@ -40,6 +40,22 @@ spec:
           env:
             - name: SERVER_NAME
               value: rt.ocf.berkeley.edu
+          livenessProbe:
+            exec:
+              command:
+              - sh
+              - '-c'
+              - '/opt/rt/healthcheck $SERVICE_SERVICE_HOST $SERVICE_SERVICE_PORT'
```

Comment:
Am I the only one who toys with the idea of writing it as

    - ['sh', '-c', '/opt/rt/healthcheck $SERVICE_SERVICE_HOST $SERVICE_SERVICE_PORT']

Reply:
I'm not actually sure if this would work, since this is creating a list of list of strings. Also, the line is too long.

Reply:
Derp, should've been `command: ['sh', '-c', '/opt/rt/healthcheck $SERVICE_SERVICE_HOST $SERVICE_SERVICE_PORT']`. But I get your point on the line length. Just looks a little weird to me written out vertically like this.